src/components/Stats/Chart/index.jsx

```diff
@@ -0,0 +1,43 @@
+/* eslint-disable react/prop-types */
```

Comment:
did importing the `listType` not work here?

Reply:
no i couldnt get the proptypes working in the file. lets go over it in the morning